tests/parse/test_parse_disease_terms.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+
+from scout.parse.disease_terms import (
+    consolidate_gene_and_hpo_annotation,
+    get_all_disease_terms,
+    get_omim_disease_terms,
+    get_orpha_disease_terms,
+    parse_disease_terms,
+)
+
+
+def test_get_all_disease_terms(
+    orphadata_en_product6_lines,
+    orphadata_en_product4_lines,
+    genemap_handle,
+    hpo_phenotype_annotation_handle,
+    test_orpha_disease_terms,
+    test_omim_disease_terms,
+):
+    #: GIVEN lines from the disease sourcefiles
+    #: WHEN the disease are compiled
+    result = get_all_disease_terms(
+        hpo_annotation_lines=hpo_phenotype_annotation_handle,
+        genemap_lines=genemap_handle,
+        orpha_to_genes_lines=orphadata_en_product6_lines,
+        orpha_to_hpo_lines=orphadata_en_product4_lines,
+    )
+    #: THEN assert correct contents are present
+    for key in test_omim_disease_terms:
+        assert key in result
+        assert test_omim_disease_terms[key]["hpo_terms"].issubset(result[key]["hpo_terms"])
+        assert test_omim_disease_terms[key]["hgnc_symbols"].issubset(result[key]["hgnc_symbols"])
+        assert test_omim_disease_terms[key]["hgnc_ids"].issubset(result[key]["hgnc_ids"])
+    for key in test_orpha_disease_terms:
+        assert key in result
+        assert test_orpha_disease_terms[key]["hpo_terms"].issubset(result[key]["hpo_terms"])
+        assert test_orpha_disease_terms[key]["hgnc_ids"].issubset(result[key]["hgnc_ids"])
+
+
+def test_parse_disease_terms(test_omim_disease_terms, test_orpha_disease_terms):
```

Comment:
This is the perfect situation where you can parametrize the test into 2 distinct tests using @pytest.mark.parametrize, alternatively I would split this tests into 2 single tests, one for OMIM and one for orpha

Reply:
The test is for a function combining orpha and omim terms and asserts the contents from both are present in the result